custom/conf/app.example.ini

```diff
@@ -1895,6 +1895,9 @@ LEVEL = Info
 ;;
 ;; Minio checksum algorithm: default (for MinIO or AWS S3) or md5 (for Cloudflare or Backblaze)
 ;MINIO_CHECKSUM_ALGORITHM = default
+;;
+;; Minio use virtual host style instead of path style when STORAGE_TYPE is `minio`
+;VIRTUAL_HOST = false
```

Comment:
I think an "enum" option could be better than the "bool" option. According to the minio's option:

```
	BucketLookupAuto BucketLookupType = iota
	BucketLookupDNS
	BucketLookupPath
```

I think we could introduce a option like this:

```
MINIO_LOOKUP_TYPE=auto
MINIO_LOOKUP_TYPE=dns
MINIO_LOOKUP_TYPE=path
```

Reply:
Good idea! I'll modify it.